### PR TITLE
Update Frontend UI

### PR DESCRIPTION
**Pull request by blackzlq**
1. check on the highest value from NLP
2. improve DLP title
3. Range bar will change color and have value showed

Tested-by: zlq on local

### CODE REVIEW

File: applications/rag/frontend/container/main.py [hunk 1]
```diff
@@ -191,7 +191,7 @@ def handlePrompt():
         error_traceback = traceback.format_exc()
         response = jsonify({
             "error": "An error occurred",
-            "message": f"Error: {err}\nTraceback:\n{error_traceback}"
+            "statusText": f"Error: {err}\nTraceback:\n{error_traceback}"
```
Comment (imreddy13) [inline applications/rag/frontend/container/main.py:194]: what does this change?

**Review by imreddy13 — APPROVED** [review]

File: applications/rag/frontend/container/main.py (hunk 1 above)
Comment (blackzlq) [reply applications/rag/frontend/container/main.py:194]: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/frontend/container/static/script.js#L80

File: applications/rag/frontend/container/static/script.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 function onReady() {
     autoResizeTextarea()
     populateDropdowns()
-    fetchDLPEnabled()
```
Comment (jindijamie) [inline applications/rag/frontend/container/static/script.js:19]: why this line is removed?

File: applications/rag/frontend/container/static/script.js [hunk 3]
```diff
@@ -142,6 +142,42 @@ function toggleNlpFilterSection(nlpEnabled) {
...
+        // Update the display text
+        valueDisplay.textContent = value;
+
+        // Determine the color based on the value
+        let color;
+        if (value <= 25) {
+            color = '#EA4335';
+        } else if (value <= 50) {
+            color = '#FBBC05';
+        } else if (value <= 75) {
```
Comment (jindijamie) [inline applications/rag/frontend/container/static/script.js:160]: can we have comments on what those colors are?

File: applications/rag/frontend/container/static/script.js (hunk 2 above)
Comment (blackzlq) [reply applications/rag/frontend/container/static/script.js:19]: moved to line 98

File: applications/rag/frontend/container/static/script.js (hunk 3 above)
Comment (blackzlq) [reply applications/rag/frontend/container/static/script.js:160]: Sure, will do in next PR

**Pull request merged**